UnityProject/Assets/Scripts/Items/Headset.cs

```diff
@@ -4,15 +4,28 @@
 /// <summary>
 ///     Headset properties
 /// </summary>
-public class Headset : NetworkBehaviour
+public class Headset : NetworkBehaviour, IInteractable<HandActivate>
 {
 	[SyncVar] public EncryptionKeyType EncryptionKey;
+	[SyncVar] public bool LoudSpeakOn = false;
```

Comment:
Worth changing these to hooks

Reply:
Only need to do that if you want to do something on client when the value is changed, unneeded here.

Reply:
dont think so

Reply:
ok